modules/acquia_cms_tour/src/Plugin/AcquiaCmsStarterKit/StarterKitSelectionForm.php

```diff
@@ -46,7 +54,7 @@ public function getFormId() {
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
 
-    $service = \Drupal::service('acquia_cms_tour.starter_kit');
+    $service = $this->starterKitService;
```

Comment:
Since we already possess a service variable, `$this->starterKitService`, I would like to understand the rationale for initializing a separate instance. Could you please clarify the necessity for this additional service variable?

```suggestion
```
@vishalkhode1 what's your opinion?

Reply:
Just a avoiding the direct drupal call as we already doing these approach in the similar form i.e StarterKitConfigForm

Reply:
I mean we don't need `$service` variable directly you can use `$this->starterKitService`.

Reply:
Done